packages/help-center/src/components/help-center-header.tsx

```diff
@@ -117,17 +118,17 @@ const ChatEllipsisMenu = () => {
 					<Icon icon={ comment } />
 					<div>{ __( 'New chat', __i18n_text_domain__ ) }</div>
 				</button>
-				<Button onClick={ handleViewChats } disabled={ totalNumberOfConversations === 0 }>
+				<button onClick={ handleViewChats } disabled={ recentConversations.length === 0 }>
```

Comment:
Since we only show the `History` button starting from two conversations, we should probably follow the same approach here:

```suggestion
				<button onClick={ handleViewChats } disabled={ recentConversations.length < 2 }>
```